spec/runtime/setup_spec.rb

```diff
@@ -1080,19 +1080,16 @@ def lock_with(ruby_version = nil)
     end
   end
 
-  describe "when Psych is not in the Gemfile", :ruby => "~> 2.2" do
-    it "does not load Psych" do
-      gemfile ""
-      ruby <<-RUBY
-        require 'bundler/setup'
-        puts defined?(Psych::VERSION) ? Psych::VERSION : "undefined"
-        require 'psych'
-        puts Psych::VERSION
-      RUBY
-      pre_bundler, post_bundler = out.split("\n")
-      expect(pre_bundler).to eq("undefined")
-      expect(post_bundler).to match(/\d+\.\d+\.\d+/)
-    end
+  it "does not load gemified standard libraries" do
+    stdlibs = %w(Psych OpenSSL)
+    install_gemfile ""
+    ruby! <<-RUBY
+      require "bundler/setup"
+      #{stdlibs.inspect}.each do |const|
+        puts Object.const_defined?(const) ? const : "undefined"
```

Comment:
(Sorry for taking so long to get back to you.) Pushed updated commit. I had to restore the psych part because of the Ruby version constraint.